apps/nextjs/src/hooks/useClerkDemoMetadata.ts

```diff
@@ -43,7 +68,7 @@ function getResult(
   // User is logged in and has completed onboarding
   return {
     isSet: true,
-    userType: Boolean(user.user.publicMetadata.isDemoUser) ? "Demo" : "Full",
+    userType: user.user.publicMetadata.labs.isDemoUser ? "Demo" : "Full",
```

Comment:
Do we always have .labs here?

Reply:
We do after calling `isDemoStatusSet`. If you think that behaviour is a bit unclear and tucked away, I can change it

Typescript knows that it's an optional key on the metadata

Reply:
Okay great